src/components/Modals/Send/utils.ts

```diff
@@ -104,15 +103,12 @@ export const handleSend = async ({
 
   const acccountMetadataFilter = { accountId: sendInput.accountId }
   const accountMetadata = selectPortfolioAccountMetadataByAccountId(state, acccountMetadataFilter)
-  const isMetaMaskDesktop = await checkIsMetaMaskDesktop(wallet)
-  const isMetaMaskImpersonator = await checkIsMetaMaskImpersonator(wallet)
+  const isMetaMaskDesktop = checkIsMetaMaskDesktop(wallet)
   if (
     fromChainId(asset.chainId).chainNamespace === CHAIN_NAMESPACE.CosmosSdk &&
     !wallet.supportsOfflineSigning() &&
-    // MM impersonators don't support Cosmos SDK chains
-    (!isMetaMaskDesktop ||
-      isMetaMaskImpersonator ||
-      (isMetaMaskDesktop && !(await checkIsSnapInstalled())))
+    // MM only supports snap things... if the snap is installed
```

Comment:
Note, the notion of MM impersonator has been removed in this PR, we don't need it anymore 🎉 checking for correct rdns is all we need